operators/vtk_renderer/vtk_renderer.hpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef VTK_OPERATOR__HPP
+#define VTK_OPERATOR__HPP
+
+#include <string>
+#include <vector>
+
+#include <vtkImageActor.h>
```

Comment:
It might be nice to hide these dependency by e.g. using PImpl (https://en.cppreference.com/w/cpp/language/pimpl). Wit the includes in the header for each app using this operator Vtk spills into the app code. Also compile time increases.
But this is just a suggestion.

Reply:
@vicentebolea on top of slower compilation, the general concern with exposing these interfaces is that end users need them as well. For most projects packages, one can choose for dev vs non-dev packages, so that dof interest. For VTK though, I'm not sure there is a distribution mechanism without the development files?

Reply:
Please find the VTK state of the vtkRendererOp moved to a nested opaque class (a pimpl idiom). Indeed it is unnecessary and inconvenient for our headers to depend in VTK. Thanks for point this out.